examples/spark-pi-py.yaml

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: "sparkoperator.k8s.io/v1alpha1"
+kind: SparkApplication
+metadata:
+  name: spark-pi-py
+  namespace: default
+spec:
+  type: Python
+  mode: cluster
+  mainApplicationFile: "local:///opt/spark/examples/src/main/python/pi.py"
+  Arguments:
+   - "10"
+  deps:
+    jars:
+      - "local:///opt/spark/examples/jars/spark-examples_2.11-2.2.0-k8s-0.5.0.jar"
+  volumes:
+    - name: "test-volume"
+      hostPath:
+        path: "/tmp"
+        type: Directory
+  driver:
+    cores: 0.1
+    image: kubespark/spark-driver-py:v2.2.0-kubernetes-0.5.0
+    coreLimit: "200m"
+    memory: "512m"
+    labels:
+      version: 2.2.0
+    serviceAccount: spark
+    volumeMounts:
```

Comment:
why do we need the same volume on driver and executor? Can we try without volume? Examples are dangerous

Reply:
actually i see thats what exists in the original pi example. ignore me. but we should put a comment that volume is not required.

Reply:
done

Reply:
i dont see the comment